docs/code/IDataViewTypeSystem.md

```diff
@@ -661,154 +605,99 @@ This chapter contains information on the C# classes used to represent column
 types. Since the IDataView type system is extensible this list describes only
 the core data types.
 
-### `ColumnType` Abstract Class
+### `DataViewType` Abstract Class
 
-The IDataView system includes the abstract class `ColumnType`. This is the
-base class for all column types. `ColumnType` has several convenience
+The IDataView system includes the abstract class `DataViewType`. This is the
+base class for all column types. `DataViewType` has several convenience
 properties that simplify testing for common patterns. For example, the
-`IsVector` property indicates whether the `ColumnType` is an instance of
+`IsVector` property indicates whether the `DataViewType` is an instance of
```

Comment:
There is no more `IsVector` property.

Reply:
More broadly, there are now no convenience properties... thanks missed this whole paragraph.